### PR TITLE
better balloon sprites

### PR DESCRIPTION
**Pull request by swankcookie**
fluff PR. Easy merge. 
balloon

### CODE REVIEW

**Comment by swankcookie** [comment]
oh god i'm so sorry

**Comment by WJohn** [comment]
Not very good at all. These are plagued with pillow shading, and the burst sprites look like you used the spray tool in paint. These also lack any highlights, balloons are actually pretty shiny, as is rubber.

**Comment by PKPenguin321** [comment]
@WJohn before you pass judgement on these, I implore you to look at the current balloon sprites

**Comment by Dorsisdwarf** [comment]
These look WORSE than the current ones which are literally the syndie team HUD icon with strings

**Comment by swankcookie** [comment]
@WJohn nowhere did i say they were good. Better. The current ones are literally red with a solid gray outline.  You are right about the highlights though, i will add to them.

**Comment by WJohn** [comment]
Oh for sure, our current balloons are trash too, but I'd rather have no shading over pillow shading.

**Comment by swankcookie** [comment]
can you specify what you mean by that?

**Comment by Dorsisdwarf** [comment]
Pillow shading is where something is "shaded" by making it uniformly darker towards all edges.

**Comment by WJohn** [comment]
Basically pillow shading does not show any sort of roundness or shape with a sprite, it looks just as flat as having no shading at all.

If you want shading, make it come from an angle, like the top, top left, or top right. Then you can do your highlights and shading according to that.

**Comment by swankcookie** [comment]
Is there a specific direction that ss13 sprites tend to be lit from?

**Comment by optimumtact** [comment]
update to latest master
git rebase -Xignore-all-space upstream/master

**Comment by Dorsisdwarf** [comment]
SS13 sprites are generally lit from above-left or above-right

File: tgstation.dme [hunk 1]
```diff
@@ -916,7 +916,10 @@
 #include "code\modules\awaymissions\mission_code\blackmarketpackers.dm"
 #include "code\modules\awaymissions\mission_code\centcomAway.dm"
 #include "code\modules\awaymissions\mission_code\challenge.dm"
+<<<<<<< HEAD
```
Comment (optimumtact) [inline tgstation.dme]: you left the conflict markers in the dme, just remove them

**Comment by swankcookie** [comment]
@dorsisdwarf ok I'll get to those, And oranges, yeah I realized right as I was going to sleep.

**Comment by swankcookie** [comment]
@Dorsisdwarf pls remove travis fail

**Comment by tkdrg** [comment]
@WJohn might want to check this again

**Comment by swankcookie** [comment]
no problem

**Comment by WJohn** [comment]
I'm still not a fan of the sprites. The harsh lighting and bumpy texture while lacking any very bright highlights makes this look closer to a very red ROCK than it does a rubber balloon.

**Comment by swankcookie** [comment]
kk. I'll fix it when i soper up.

**Pull request closed without merging**